opcua/server/address_space.py

```diff
@@ -180,34 +180,46 @@ def add_nodes(self, addnodeitems, user=User.Admin):
     def _add_node(self, item, user):
         result = ua.AddNodesResult()
 
-        if item.RequestedNewNodeId in self._aspace:
+        # Check if the null NodeId is given
+        if item.RequestedNewNodeId.is_null():
+            self.logger.debug("RequestedNewNodeId is null")
+            if item.BrowseName.NamespaceIndex:
```

Comment:
OK. null NodeId has 0 as idx. so I propose to use the namespace of the parent, not the namespace of the browsename.
And if parent is Null, return BadParentNodeIDInvalid. We should not allow creation of node without valid parent. Then I am fine with PR